polytracker/src/polytracker/main.cpp

```diff
@@ -145,19 +145,19 @@ void polytracker_end() {
 }
 
 void polytracker_print_settings() {
-  for (auto target_source : get_target_sources()) {
-    printf("POLYPATH:      %s\n", target_source.c_str());
-  }
-  if (get_target_sources().empty()) {
-    printf("POLYPATH:      *\n");
-  }
+  // for (auto target_source : get_target_sources()) {
```

Comment:
Why not remove it, instead of commenting it out? If we need to get it back there's always the git history, and it would remove a bunch of noise from the source code
(same for all other commented-out lines)